src/pybind/mgr/dashboard/frontend/src/app/shared/components/progress/progress.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="d-flex flex-column justify-content-center align-items-center bold">
+  <ng-container>
+    <h3 class="text-center"
+        [ngClass]="{ 'mt-3': inProgress && isPaused }">
+      <i *ngIf="inProgress && isPaused; else spinningIcon"
```

Comment:
if we see having both spinner and progress now is creating redundancy.

Reply:
Should be a part of carbonization efforts once we take it up